runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -326,6 +327,17 @@ public static TrayIcon getTrayIcon()
 		return trayIcon;
 	}
 
+	public void notify(String message)
+	{
+		notify(message, TrayIcon.MessageType.NONE);
```

Comment:
There should be a modification to the animation idle plugin in another commit to use this